.github/workflows/dialogflow-cx-python.yaml

```diff
@@ -0,0 +1,173 @@
+          pip install -r requirements-test.txt
+      - id: 'auth'
+        name: 'Authenticate to GCP'
+        uses: 'google-github-actions/auth@v0'
+        with:
+            create_credentials_file: 'true'
+            workload_identity_provider: 'projects/792212140555/locations/global/workloadIdentityPools/gh-pool-df-3/providers/gh-provider-df-3'
+            service_account: 'oidc-sa-df-3@df-terraform-dev.iam.gserviceaccount.com'
+      - name: Dowload UUID
+        uses: actions/download-artifact@v1
+        with:
+          name: shared
+      - name: UUID to env
+        id: build_id
+        run: |-
+          cat ../shared/uuid.txt
+      - name: Run unit tests
+        run: |-
+          pytest -m hermetic
+      - name: Run integration tests
+        run: |-
+          pytest -m integration
+        env:
+          PROJECT_ID: df-terraform-dev
+          BUILD_UUID: ${{ steps.build_id.outputs.id }}
+          GOOGLE_APPLICATION_CREDENTIALS: ${{ steps.auth.outputs.credentials_file_path }}
+  teardown_terraform:
+    needs: [setup_terraform, integration_tests]
+    if: ${{ always() }}
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ./dialogflow-cx
+    permissions:
+      contents: 'read'
+      id-token: 'write'
+    steps:
+      - uses: actions/checkout@v3
+      - id: 'auth'
+        name: 'Authenticate to GCP'
+        uses: 'google-github-actions/auth@v0'
+        with:
+            create_credentials_file: 'true'
+            workload_identity_provider: 'projects/792212140555/locations/global/workloadIdentityPools/gh-pool-df-3/providers/gh-provider-df-3'
+            service_account: 'oidc-sa-df-3@df-terraform-dev.iam.gserviceaccount.com'
+      - name: Dowload UUID
+        uses: actions/download-artifact@v1
+        with:
+          name: shared
+      - name: UUID to env
+        id: build_id
+        run: |-
+          cat ../shared/uuid.txt
+      - id: 'basic_sample'
+        name: Get Webhook Env Vars
+        run: |-
+          export WEBHOOK_NAME
+          export WEBHOOK_ENTRYPOINT
+          WEBHOOK_NAME="$(python -c "from webhook.main import get_webhook_name; print(get_webhook_name(\"${BUILD_UUID?}\"))")"
+          WEBHOOK_ENTRYPOINT="$(python -c "from webhook.main import get_webhook_entrypoint; print(get_webhook_entrypoint())")"
+          echo "::set-output name=webhook-name::$WEBHOOK_NAME"
+          echo "::set-output name=webhook-entrypoint::$WEBHOOK_ENTRYPOINT"
+        env:
+          PROJECT_ID: df-terraform-dev
+          BUILD_UUID: ${{ steps.build_id.outputs.id }}
+      - name: Terraform Init/Destroy
+        run: |-
+          cp "${{ steps.auth.outputs.credentials_file_path }}" terraform/application_default_credentials.json
+          cp -r webhook terraform
+          docker run --env GOOGLE_APPLICATION_CREDENTIALS=/app/application_default_credentials.json -w /app -v "$(pwd)"/terraform:/app -v "$(pwd)"/webhook:/app/webhook hashicorp/terraform:1.1.7 init
```

Comment:
The Terraform container version 1.1.7 could be another variable, as it's mentioned in at least a couple places ... and would make updates easier

Reply:
Done.